test/models/model_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ModelTest < ActiveSupport::TestCase
```

Comment:
this class can be deleted for the moment

Reply:
Class deleted 
![removeTest](https://user-images.githubusercontent.com/58991676/71357979-23754000-2566-11ea-92ab-946f646f2d56.png)